readthedocs/templates/builds/build_detail.html

```diff
@@ -161,6 +161,13 @@ <h3>{% trans "Error" %}</h3>
            data-bind="text: error">
           {{ build.error }}
         </p>
+        <p>
+          {% block github_issue_link %}
+            {% blocktrans with url=issue_url %}
+              Report any build issues <a href="{{ url }}">here</a>.
+            {% endblocktrans %}
+          {% endblock %}
+        </p>
```

Comment:
This looks good so far! However, we don't want to unconditionally show this paragraph block on a build error, we only want to show this if there was an *unknown* error with the build -- otherwise we get swamped with support requests we can't handle. We don't have a great way of determining this from our build database yet.

We set a generic build failure here:
https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/projects/tasks.py#L303-L308

I don't have any immediate suggestions on how to determine here if the build was a generic failure though. Let me think about this more, or perhaps someone else has a suggestion for the interim.

I started to outline what I'm describing in #3399, but this is a large amount of work we haven't decided on yet.

Reply:
I would like to take up #3399 once we decide the method to determine generic build failure. We could add a field to build models which stores the failure type and then use that to show the paragraph block.

Reply:
I push some changes that checks for the generic error and if it's not, the link is not shown.